flashlight/nn/Utils.h

```diff
@@ -13,13 +13,38 @@
 
 #pragma once
 
+#include <iomanip>
+
 #include <af/dim4.hpp>
 
 #include "flashlight/common/Defines.h"
+#include "flashlight/common/Utils.h"
 #include "flashlight/nn/modules/Module.h"
 
+#ifdef TRACE_PRECISION
```

Comment:
Can we put this in an anonymous namespace?

Reply:
Sure.